src/main/resources/META-INF/frontend/fc-month-calendar/vaadin-month-calendar.css

```
@@ -0,0 +1,55 @@
+ */
+:host([theme~="fc-month-calendar"]) {
+  
+  td, th {
+    width: calc(100% / 7);
+  }
+
+  [part~='date'][part~='selected']::before {
+    background: var(--vaadin-date-picker-date-selected-background, var(--vaadin-text-color, var(--_selection-color)));
+  }
+
+  [part~='date']:not(:empty):not([part~='disabled']):not([part~='selected']):hover::before {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)